api/src/main/java/io/kafbat/ui/controller/KafkaConnectController.java

```diff
@@ -285,4 +285,26 @@ private Comparator<FullConnectorInfoDTO> getConnectorsComparator(ConnectorColumn
       default -> defaultComparator;
     };
   }
+
+  @Override
+  public Mono<ResponseEntity<Void>> resetConnectorOffsets(String clusterName, String connectName,
```

Comment:
Can we add some tests for the new functionality? 
`KafkaConnectServiceTests` looks like a good place to start

Reply:
Sure, I'll look into that